src/features/click-to-play.js

```diff
@@ -1029,10 +1355,10 @@ class DuckWidget {
     }
 }
 
-async function initCTL (resp) {
-    for (const entity of Object.keys(config)) {
+async function initCTL (configResponseData) {
```

Comment:
Nit: I get renaming stuff but since we are both touching this code, it would be good to keep this to a minimum as it introduces merge conflicts

Reply:
👍 ✅